command/polly_tts.js

```diff
@@ -38,7 +38,7 @@ var polly_tts = {
       let params = {
         Text: ssmlText,
         OutputFormat: 'mp3',
-        SampleRate: '16000',
+        SampleRate: '22050',
```

Comment:
Should make the change in `constants.samplerate.SAMPLERATE_MP3` and pick it up from there (so the DB records will agree with the value used in file generation).

Reply:
This is just for debugging so they can try it out.  I'm going to close this one anyways because I somehow included some extra stuff in package.json.